.evergreen/integration-tests.sh

```diff
@@ -61,6 +61,10 @@ if [ -z "$ORCHESTRATION_FILE" ]; then
    if [ "$SSL" != "nossl" ]; then
       ORCHESTRATION_FILE="${ORCHESTRATION_FILE}-ssl"
    fi
+
+   if [ -n "$LOAD_BALANCER" ]; then
```

Comment:
I'm not super excited about using `-n` to test a variable that is set with the value `true` in `config.yml`.  This seems to create the possibility that someone could think that setting `LOAD_BALANCER=false` would disable load balancer tests, but that would clearly not happen.  I can't decide on a clearly better approach, but what about testing for `= "on"` as is done for `IPV4_ONLY`?  The the `on/off` semantics seem a bit more natural.

Reply:
Agreed; suggest using same pattern as `$IPV4_ONLY` check above:
```bash
if [ "$LOAD_BALANCER" = "on" ]; then
```
Also suggest adding `LOAD_BALANCER=${LOAD_BALANCER:-off}` above to document script parameters and default behavior.

Note: the `VAR=${VAR:-value}` pattern used above can be rewritten as:
```bash
: "${VAR:=value}"
```

Reply:
> This seems to create the possibility that someone could think that setting LOAD_BALANCER=false would disable load balancer tests, but that would clearly not happen.

Good point. Updated to check for the value "on".

> Note: if this script is unconditionally executed in Bash, the VAR=${VAR:-value} pattern used above can be rewritten

Thank you for the suggestion. Updated to explicitly run with bash and declare the `LOAD_BALANCER` variable.